packages/matplotlib/meta.yaml

```diff
@@ -25,6 +25,7 @@ build:
   cflags: -s USE_FREETYPE=1 -s USE_LIBPNG=1 -s USE_ZLIB=1
   ldflags: ../../../../emsdk/emsdk/.emscripten_cache/asmjs/libpng.bc
   post: |
+    wget http://antiyawn.com/uploads/Humor-Sans-1.0.ttf && mv Humor-Sans-1.0.ttf $SITEPACKAGES/matplotlib/mpl-data/fonts/ttf/Humor-Sans.ttf
```

Comment:
Maybe directly save it to the right location with `-O` flag?

Reply:
Aah yes, let me change and try a quick rebuild of ``matplotlib`` and then update this PR.